src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -419,13 +419,22 @@ def list_app_service_plans(resource_group_name=None):
     return plans
 
 
+def _linux_sku_check(sku):
+    tier = _get_sku_name(sku)
+    if tier == 'BASIC' or tier == 'STANDARD':
```

Comment:
on a second thought, you should do  `!= 'FREE'`, this way if new sku comes up, you don't need to change the code